### DISCUSSION TITLE
Links para las sesiones

### DISCUSSION THREAD

**Discussion by DSBC26** [Launch-X-Latam/OnBoarding#13] [Q&A]
Hola, una pregunta donde puedo hallar los links de las sesiones para conectarme en vivo?, he buscado y no los encuentro

**Comment by devonent** [comment]
Los links de los lives de esta primera etapa se irán publicando en la sección de "Publicaciones" dentro del equipo "Launch X: Misión FrontEnd"

<div align="center">
  <img src="https://user-images.githubusercontent.com/87509978/192679531-12051c2b-3059-4ab8-947f-b6e6d4c56cbe.png" width="500px">
</div>

Aún no han confirmado nada pero, probablemente (debido a ciclos pasados como Operación Aleph) se estará creando un canal por cada semana, en dónde se publicarán los links de los respectivos lives. Los canales los puedes encontrar en el panel izquierdo:

<div align="center">
   <img src="https://user-images.githubusercontent.com/87509978/192680039-aae0b572-5f13-484d-b91a-54969767b4f3.png" width="300px">
</div>

Si no puedes asistir a algun live, en la parte superior hay una sección llamada "Grabaciones" en dónde podrás acceder a las grabaciones de los lives. Probablemente tarden en subirse un poco los lives, pero no suele pasar más de uno o dos días desde el día de transimisión.

<div align="center">
    <img src="https://user-images.githubusercontent.com/87509978/192680324-af57133f-c5ea-487a-83da-4abbbd8b6031.png" width="500px">
</div>
